docs/lightning/03-cti-adapter/09-recording-controls.md

```diff
@@ -39,9 +39,9 @@ This feature can sometimes cause issues if an existing contact flow manages call
 
 ## Recording Named Credential
 
-Starting with version v5.22, you now have an area in the CTI Adapter to specify the recording Named Credential you wish to use with the adapter. Follow the instructions in the "Setup" section above to create the Named Credentials. Once you have the names for the Named Credentials, add them directly to the Adapters you wish to use.
+Starting with version v5.23.3, you now have an area in the CTI Adapter to specify the recording Named Credential you wish to use with the adapter. Follow the instructions in the "Setup" section above to create the Named Credentials. Once you have the names for the Named Credentials, add them directly to the Adapters you wish to use.
 
-**CTI Adapter Named Credential Location:**
+#### CTI Adapter Named Credential Location:
```

Comment:
Since you have to fix the change in the release notes can you change this to use h3 ("###")?

Reply:
Not sure what you mean. DIdn't you approve this in staging repo?

Reply:
As I responded there, kept to h4 inorder to make it size of header

Reply:
> DIdn't you approve this in staging repo?

I shouldn't have. Based on this: https://www.w3.org/WAI/tutorials/page-structure/headings/

"Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a h2 is not followed directly by an h4"